PRODUCTIONISATION.md

```diff
@@ -64,11 +70,10 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 
 * 5.1 Continuous Delivery pipeline configured to:
   * [ ] 5.1.1 Automatically test pull requests
-  * [ ] 5.1.2 `master` branch to run linting and tests
+  * [ ] 5.1.2 `master` branch to run lint (_if applicable_) and tests
```

Comment:
best to split this so scoring can be applied appropriately:

```suggestion
  * [ ] 5.1.2 `master` branch to run code checks
    * [ ] 5.1.2.1 Code Standard checks (linting)
    * [ ] 5.1.2.2 Tests
```


Very wary of using wording such as "where applicable", or "where possible". This should instead be reflected in reports generated, where it's one of "done", "not done", "not applicable - <justification>"